discretisedfield/field.py

```diff
@@ -662,12 +663,82 @@ def mean(self, axis=None):
         >>> field.mean()
         array([55.])
 
+        3. Computing the vector field mean along x direction.
+
+        >>> import discretisedfield as df
+        ...
+        >>> p1 = (0, 0, 0)
+        >>> p2 = (5, 5, 5)
+        >>> cell = (1, 1, 1)
+        >>> mesh = df.Mesh(p1=p1, p2=p2, cell=cell)
+        ...
+        >>> field = df.Field(mesh=mesh, nvdim=3, value=(0, 0, 1))
+        >>> field.mean(direction='x')((2.5, 0.5, 0.5))
+        array([0., 0., 1.])
+
+        4. Computing the vector field mean along x and y directions.
+
+        >>> import discretisedfield as df
+        ...
+        >>> p1 = (0, 0, 0)
+        >>> p2 = (5, 5, 5)
+        >>> cell = (1, 1, 1)
+        >>> mesh = df.Mesh(p1=p1, p2=p2, cell=cell)
+        ...
+        >>> field = df.Field(mesh=mesh, nvdim=3, value=(0, 0, 1))
+        >>> field.mean(direction=['x', 'y'])((2.5, 2.5, 0.5))
```

Comment:
Do we want a list of strings or just one string with multiple elements such as `'xy'`? I know that the latter would cause problems with multi-character direction names. However, we currently have the same limitations in other parts of the code.